mmv1/products/apphub/Workload.yaml

```diff
@@ -0,0 +1,241 @@
+# Copyright 2024 Google Inc.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+--- !ruby/object:Api::Resource
+base_url: projects/{{project}}/locations/{{location}}/applications/{{applications_id}}/workloads
+create_url: projects/{{project}}/locations/{{location}}/applications/{{applications_id}}/workloads?workloadId={{workload_id}}
+self_link: projects/{{project}}/locations/{{location}}/applications/{{applications_id}}/workloads/{{workload_id}}
+id_format: projects/{{project}}/locations/{{location}}/applications/{{applications_id}}/workloads/{{workload_id}}
+import_format:
+  - projects/{{project}}/locations/{{location}}/applications/{{applications_id}}/workloads/{{workload_id}}
+name: Workload
+description: 'Workload represents a binary deployment (such as Managed Instance Groups (MIGs), GKE deployments, etc.) that performs the smallest logical subset of business functionality.
+  It registers identified workload to the Application.'
+autogen_async: true
+custom_code: !ruby/object:Provider::Terraform::CustomCode
+  constants: 'templates/terraform/constants/apphub_discovered_resource.go.erb'
+examples:
+  - !ruby/object:Provider::Terraform::Examples
+    name: "workload_basic"
+    pull_external: true
+    primary_resource_id: "example"
+    config_path: "templates/terraform/examples/apphub_workload_basic.tf.erb"
```

Comment:
Nit: if you use `name: "apphub_workload_basic"` you should be able to remove `config_path`